contracts/TransferLimitModule.sol

```diff
@@ -0,0 +1,417 @@
+
+        // Perform transfer by invoking manager
+        if (token == 0) {
+            require(manager.execTransactionFromModule(to, amount, "", Enum.Operation.Call), "Could not execute ether transfer");
+        } else {
+            bytes memory data = abi.encodeWithSignature("transfer(address,uint256)", to, amount);
+            require(manager.execTransactionFromModule(token, 0, data, Enum.Operation.Call), "Could not execute token transfer");
+        }
+
+        // We transfer the calculated tx costs to the tx.origin to avoid sending it to intermediate contracts that have made calls
+        if (gasPrice > 0) {
+            handlePayment(startGas, dataGas, gasPrice, gasToken, refundReceiver);
+        }
+    }
+
+    /// @dev Returns hash to be signed by owners.
+    /// @param token Address of the token that should be transfered (0 for Ether)
+    /// @param to Address to which the tokens should be transfered
+    /// @param amount Amount of tokens (or Wei) that should be transfered
+    /// @param _nonce Nonce used for this Safe transaction.
+    /// @param safeTxGas Gas that should be used for the Safe transaction.
+    /// @param dataGas Gas costs for data used to trigger the safe transaction and to pay the payment transfer
+    /// @param gasPrice Gas price that should be used for the payment calculation.
+    /// @param gasToken Token address (or 0 if ETH) that is used for the payment.
+    /// @param refundReceiver Address of receiver of gas payment (or 0 if tx.origin).
+    /// @param _nonce Transaction nonce.
+    /// @return Transaction hash.
+    function getTransactionHash(
+        address token,
+        address to,
+        uint256 amount,
+        uint256 safeTxGas,
+        uint256 dataGas,
+        uint256 gasPrice,
+        address gasToken,
+        address refundReceiver,
+        uint256 _nonce
+    )
+        public
+        view
+        returns (bytes32)
+    {
+        return keccak256(
+            abi.encodePacked(byte(0x19), byte(0), this, token, to, amount, safeTxGas, dataGas, gasPrice, gasToken, refundReceiver, _nonce)
+        );
+    }
+
+    /// @dev Returns start of the current time period.
+    /// @return Unix timestamp.
+    function currentStartTime()
+        public
+        view
+        returns (uint)
+    {
+        return getNow() - (getNow() % timePeriod);
+    }
+
+    function handleTransferLimits(address token, uint256 amount)
+        internal
+        returns (bool)
+    {
+        TransferLimit storage transferLimit = transferLimits[token];
+        // If time period is over, reset expenditure.
+        if (isPeriodOver()) {
+            transferLimit.spent = 0;
+            totalWeiSpent = 0;
+            totalDaiSpent = 0;
+        }
+
+        // Transfer + previous expenditure shouldn't exceed limit specified for token.
+        if (SafeMath.add(transferLimit.spent, amount) > transferLimit.transferLimit) {
```

Comment:
Can you use `SafeMath` via the `using` statemaent (https://github.com/gnosis/safe-contracts/blob/development/contracts/GnosisSafe.sol#L15)